agent-bootstrapper/src/com/thoughtworks/go/agent/bootstrapper/AgentBootstrapper.java

```diff
@@ -46,24 +48,19 @@ public AgentBootstrapper(AgentLauncherCreator launcherCreator) {
         this.launcherCreator = launcherCreator;
     }
 
-    public static void main(String[] args) {
-        BootstrapperLoggingHelper.initLog4j();
```

Comment:
This used to setup logging for agent-bootstrapper. Is getting rid of this intentional?